### PR TITLE
pipelines: outputs: kafka: Document the raw_log_key

### PR DESCRIPTION
**Pull request by zecke**
This is similar to the log_key in other outputs, e.g. the cloudwatch one.

### CODE REVIEW

File: pipeline/outputs/kafka.md [hunk 1]
```diff
@@ -114,3 +115,28 @@ specific avro schema.
     rdkafka.log_level 7
     rdkafka.metadata.broker.list 192.168.1.3:9092
 ```
+
+#### Kafka Configuration File with Raw format
+
+This example fluent-bit config creates dummy records with the _payloadkey_ and
+_msgkey_ keys. The value of the _msgkey_ will be used as the Kafka message key
+and the value of the _payloadkey_ as payload.
```
Comment (lockewritesdocs) [inline pipeline/outputs/kafka.md:123]:
```suggestion
This example Fluent Bit configuration file creates dummy records with the
_payloadkey_ and _msgkey_ keys. The _msgkey_ value is used as the Kafka message
key, and the _payloadkey_ value as the payload.
```

File: pipeline/outputs/kafka.md [hunk 2]
```diff
@@ -114,3 +115,28 @@ specific avro schema.
...
+
+
+```text
+[INPUT]
+    Name dummy
+    Tag  dummy.data
```
Comment (lockewritesdocs) [inline pipeline/outputs/kafka.md:129]:
```suggestion
    Name example
    Tag  example.data
```

Comment (lockewritesdocs) [reply pipeline/outputs/kafka.md:129]: "dummy" is not an inclusive term, and can be seen as a pejorative, so we avoid those words where possible.

**Review by lockewritesdocs — CHANGES_REQUESTED** [review]
Thanks for your submission! I suggested some editorial changes to help align with Chronosphere tone and style.

File: pipeline/outputs/kafka.md (hunk 2 above)
Comment (patrick-stephens) [reply pipeline/outputs/kafka.md:129]: @lockewritesdocs this is referring to the dummy input so the `name` has to be that: https://docs.fluentbit.io/manual/pipeline/inputs/dummy

File: pipeline/outputs/kafka.md [hunk 3]
```diff
@@ -114,3 +115,28 @@ specific avro schema.
     rdkafka.log_level 7
     rdkafka.metadata.broker.list 192.168.1.3:9092
 ```
+
+#### Kafka Configuration File with Raw format
+
+This example Fluent Bit configuration file creates dummy records with the
```
Comment (lockewritesdocs) [inline pipeline/outputs/kafka.md:121]:
```suggestion
This example Fluent Bit configuration file creates example records with the
```

Comment (lockewritesdocs) [reply pipeline/outputs/kafka.md:121]: Apologies for not catching that instance of "dummy" earlier.

**Review by lockewritesdocs — APPROVED** [review]
Thanks for incorporating changes! LGTM 🦖

**Pull request merged**